### DISCUSSION TITLE
谁能把重庆市大渡口区 恒大麓山湖 停贷消息加上

### DISCUSSION THREAD

**Discussion by hzy459176895** [WeNeedHome/SummaryOfLoanSuspension#446] [General]
谁能把重庆市大渡口区 恒大麓山湖 停贷消息加上

**Comment by leonce28** [comment]
给停贷图片或者通知书

**Reply by hzy459176895** [reply]
嗯 我发了一个

**Reply by hzy459176895** [reply]
大佬！！！ 栓Q~ 哈哈哈

**Reply by hzy459176895** [reply]
![11111](https://user-images.githubusercontent.com/67032748/179153668-f5c34ca0-5e92-4dba-b466-9d4d60f88290.png)

**Reply by JadyLiu98** [reply]
请问可以怎么联系上业主呀

**Reply by liusshuai** [reply]
我也买了大渡口，买的佳兆业，你们是真的有打算停贷吗

**Reply by JadyLiu98** [reply]
我在帮一个荷兰广播电视协会电视部(NOS)记者找人采访。有人想聊一下情况吗？这个是媒体的网址：www.nos.nl

**Reply by Vucius** [reply]
> 我在帮一个荷兰广播电视协会电视部(NOS)记者找人采访。有人想聊一下情况吗？这个是媒体的网址：www.nos.nl

你觉得他们还不够惨么？你找开发商销售广告和现场实拍就够写文章的了。别拉他们下水。

**Reply by JadyLiu98** [reply]
> > 我在帮一个荷兰广播电视协会电视部(NOS)记者找人采访。有人想聊一下情况吗？这个是媒体的网址：www.nos.nl
> 
> 你觉得他们还不够惨么？你找开发商销售广告和现场实拍就够写文章的了。别拉他们下水。

你可能没有区分报道和软文
